bin/create_svg_manifest.js

```diff
@@ -0,0 +1,32 @@
+const glob = require('glob');
+const _ = require('lodash');
+const fs = require('fs'); // eslint-disable-line
+
+function reducer(acc, curr) {
+  const camel = curr.split('.')[0];
+
+  return Object.assign({}, acc, {
+    [_.camelCase(camel)]: `./${curr}`
+  });
+}
+
+function generateManifest(err, matches) {
+  const manifest = matches.reduce(reducer, {});
```

Comment:
can you think of a more descriptive name for the reducer? since it's the function argument to `.reduce`, we know it's a reducer, so the method name doesn't give us an additional information